src/react/components/common/errorHandler/errorHandler.tsx

```diff
@@ -77,6 +77,14 @@ export class ErrorHandler extends React.Component<IErrorHandlerProps> {
      * @param error The error to handle
      */
     private handleError(error: string | Error | AppError) {
+        // This is a special case where we don't need to throw an
```

Comment:
Is there no way for us to catch this within the tags input component?  Fix the underlying cause?

Reply:
It's thrown by the ReactDnD inside the tags input component we're using... My fix is the new tags input component. This is just the patch for the time being